tree/ntuple/v7/test/CustomStruct.hxx

```diff
@@ -208,4 +209,17 @@ class Cyclic {
    std::vector<Cyclic> fMember;
 };
 
+class Left {
+public:
+   float x = 1.0;
+   virtual ~Left() = default;
+   ClassDef(Left, 1)
+};
+
+class DerivedFromLeftAndTObject : public Left, public TObject {
+public:
+   virtual ~DerivedFromLeftAndTObject() = default;
+   ClassDef(DerivedFromLeftAndTObject, 1)
```

Comment:
I believe this should be necessary for things to work as expected right?
```suggestion
   ClassDefOverride(DerivedFromLeftAndTObject, 1)
```

Reply:
Since there is no other function, it should make no practical difference.